papers/valentina_staneva/myst.yml

```
@@ -0,0 +1,52 @@
+version: 1
+project:
+  # Update this to match `scipy-2024-<folder>` the folder should be `<firstname_surname>`
+  id: scipy-2024-valentina_staneva
+  # Ensure your title is the same as in your `main.md`
+  title: "echodataflow: Configurable, Reproducible, and Scalable Fisheries Acoustics Workflows with Prefect"
+  subtitle: MyST Markdown edition
+  # Authors should have affiliations, emails and ORCIDs if available
+  authors:
+    - name: Valentina Staneva
+      email: vms16@uw.edu
+      affiliations:
+        - eScience Institute, University of Washington
+      corresponding: true
+    - name: Soham Butala
+      email: sbutala@uw.edu
+      affiliations:
+        - eScience Institute, University of Washington
+    - name: Landung (Don) Setiawan
+      email:landungs@uw.edu
```

Looks like another problem in YAML (note the space):

```suggestion
      email: landungs@uw.edu
```